RELEASE.md

```diff
@@ -0,0 +1,18 @@
+Release type: major
```

Comment:
```suggestion
Release type: minor
```

Hol' up, we don't want a 1.0 release just yet :grin:

Reply:
@patrick91 is there a way we can add a safety mechanism to the release action to prevent 1.0 releases just yet, in case something like this gets through by accident?

Reply:
Whoops 😅